src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,7 +16,11 @@
 #include <string>
 #include <string.h>
 #include <sys/stat.h>
-#ifdef HAVE_SYS_SYSCTL_H
```

Comment:
Nope. Those were done by @jasonwilliams200OK. I don't have anything to do with those, and I'll just revert the changes I did to make sure I don't ruin anyone else changes :)